packages/front-end/src/components/dashboard/OptionsTable/hooks.tsx

```
@@ -220,11 +220,13 @@ const usePositions = () => {
 
           // Check state to see if the series is disabled.
           const seriesData =
-            chainData[expiryTimestamp][humanisedStrikePrice][putOrCall];
-          const buyDisabled =
-            seriesData.buy.disabled || !seriesData.buy.quote.quote;
-          const sellDisabled =
-            seriesData.sell.disabled || !seriesData.sell.quote.quote;
+            chainData?.[expiryTimestamp]?.[humanisedStrikePrice]?.[putOrCall];
```

I'd rather we didn't just ham-fist a bunch of optional chaining here as it could hide other errors. I suspect that `chainData[expiryTimestamp]` is undefined because it's trying to load positions for expiries in the past. A check on `chainData[expiryTimestamp]` for validity should be sufficient.